test/Microsoft.ApplicationInsights.AspNetCore.Tests/Extensions/ApplicationInsightsExtensionsTests.cs

```diff
@@ -237,6 +237,7 @@ public static void RegistersTelemetryClientToGetTelemetryConfigurationFromContai
                 IServiceProvider serviceProvider = services.BuildServiceProvider();
                 var configuration = serviceProvider.GetTelemetryConfiguration();
                 configuration.InstrumentationKey = Guid.NewGuid().ToString();
+                ITelemetryChannel oldChannel = configuration.TelemetryChannel;
```

Comment:
We attribute the tests to prevent them from running in parallel because of shared infrastructure stuff kind of like this.